src/ray/gcs/gcs_server/gcs_node_manager.cc

```diff
@@ -165,6 +165,7 @@ GcsNodeManager::GcsNodeManager(boost::asio::io_service &main_io_service,
             });
           })),
       node_failure_detector_service_(node_failure_detector_io_service),
+      light_heartbeat_enabled_(RayConfig::instance().light_heartbeat_enabled()),
```

Comment:
light_heartbeat_enabled_ is not need, we can use RayConfig::instance().light_heartbeat_enabled() directly.

Reply:
ok